index.html

```diff
@@ -84,197 +46,249 @@ <h4>Zero-ops OpenStack on just about any Linux box.</h4>
-            <button class="p-code-copyable__action">Copy to clipboard</button>
-          </div>
+      <a id="tab-three" href="#tab-three__content"
+        class="col-small-2 col-medium-2 col-4 p-card p-tab u-align--center u-vertically-center js-tab" role="tab">
+        <img src="https://assets.ubuntu.com/v1/236f314d-macos.svg" style="height: 145px;" alt="macOS">
+      </a>
+    </div>
 
-          <h3>Configure networks and OpenStack databases</h3>
+    <div class="row">
+      <div class="col-6 js-tab__content" id="tab-one__content" role="tabpanel" aria-labelledby="tab-one">
+        <ol class="p-stepped-list u-no-margin--left">
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">Install the microkstack snap</h3>
 
-          <div class="p-code-copyable">
-            <input class="p-code-copyable__input" aria-label="sudo microstack.init" value="sudo microstack.init --auto" readonly="readonly">
-            <button class="p-code-copyable__action">Copy to clipboard</button>
-          </div>
+            <div class="p-code-copyable u-stepped-list-margin">
+              <input class="p-code-copyable__input" aria-label="sudo snap install microk8s --classic"
+                value="sudo snap install microstack --classic --edge" readonly="readonly">
+              <button class="p-code-copyable__action">Copy to clipboard</button>
+            </div>
 
-          <p>
-            That’s it! Your OpenStack is ready.
-          </p>
-        </div>
+            <p class="p-stepped-list__content u-stepped-list-margin">Don’t have the <code>snap</code> command? <a
+                href="https://docs.snapcraft.io/installing-snapd" class="p-link--external">Get set up for snaps.</a></p>
+          </li>
+        </ol>
+        <p>
+          That’s it! Your OpenStack is ready.
+        </p>
       </div>
-      <div class="u-fixed-width">
-        <hr class="p-separator" />
+      <div class="col-6 js-tab__content" id="tab-two__content" role="tabpanel" aria-labelledby="tab-two">
+        <ol class="p-stepped-list u-no-margin--left">
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">Install Multipass for Windows</h3>
+
+            <p class="p-stepped-list__content u-stepped-list-margin">
+              <a href="https://github.com/CanonicalLtd/multipass/releases" class="p-button--positive js-download"
+                data-os="win-win">
+                <span class="p-link--external">Download Multipass for Windows</span>
+              </a>
+            </p>
+          </li>
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">Start a multipass vm</h3>
+
+            <div class="p-code-copyable u-stepped-list-margin">
+              <input class="p-code-copyable__input" aria-label="sudo snap install microk8s --classic"
```

Comment:
```suggestion
              <input class="p-code-copyable__input"
```
Again, snap name aside, that is not a label.